lang/levels/standard/text_fre.po

```diff
@@ -0,0 +1,841 @@
+
+#. Shores of Battle (lv456)
+#: guitext:80
+msgctxt "In-game message"
+msgid "It's now or never? Kill or be killed! "
+msgstr "C'est maintenant ou jamais ! Tuer ou être tué ! "
+
+#. Shores of Battle (lv456)
+#: guitext:81
+msgctxt "In-game message"
+msgid "Here they come! "
+msgstr "Les voilà ! "
+
+#. Shores of Battle (lv456)
+#: guitext:82
+msgctxt "In-game message"
+msgid "I give your performance 2 out of 5 spleens. But I guess it's the result that counts. Victory"
+msgstr "Je donne à votre performance une note de 2 organes sur 5. Mais je suppose que c'est le résultat qui compte. Victoire"
+
+#. My Pet Dungeon (lv457)
+#: guitext:83
+msgctxt "In-game message"
+msgid "Welcome to SimDungeon. You can attract 25 evil creatures per portal. Heroes can be summoned at the spawner"
+msgstr "Bienvenue dans Donjon-Simulation. Vous pouvez attirer 25 créatures maléfiques par portail. Les héros peuvent être invoqués au générateur"
+
+#. My Pet Dungeon (lv457)
+#: guitext:84
+msgctxt "In-game message"
+msgid "Find a way to build a bridge. It will help you to expand in this realm. "
+msgstr "Trouvez un moyen de construire un pont. Cela vous aidera à vous développer dans ce domaine. "
+
+#. My Pet Dungeon (lv457)
+#: guitext:85
+msgctxt "In-game message"
+msgid "You're well on your way to an impressive dungeon? Get to one hundred units and we'll talk. "
+msgstr "Vous êtes bien parti pour un donjon impressionnant ? Arrivez à cent unités et nous en reparlerons. "
+
+#. My Pet Dungeon (lv457)
+#: guitext:86
+msgctxt "In-game message"
+msgid "There is a way to breach the white wall and take over the north-west corner of the map. Useful but optional. "
+msgstr "Il existe un moyen de percer le mur blanc et de prendre le contrôle du coin nord-ouest de la carte. Utile mais facultatif. "
+
+#. My Pet Dungeon (lv457)
+#: guitext:87
+msgctxt "In-game message"
+msgid "You have a most impressive army. When you're done sell your prison. It will trigger an endless invasion. "
+msgstr "Vous avez une armée des plus impressionnantes. Quand vous aurez fini, vendez votre prison. Cela déclenchera une invasion sans fin. "
+
+#. My Pet Dungeon (lv457)
+#: guitext:88
+msgctxt "In-game message"
+msgid "Let's see how good your dungeon really is. How long can you survive? "
+msgstr "Voyons à quel point votre donjon est vraiment bon. Combien de temps pouvez-vous survivre? "
+
+#. My Pet Dungeon (lv457)
+#: guitext:89
+msgctxt "In-game message"
+msgid "You've done the impossible. Victory! "
+msgstr "Vous avez fait l'impossible. Victoire! "
+
+#. Ghost in the Machine (lv458)
+#: guitext:90
+msgctxt "In-game message"
+msgid "You have an old version of KeeperFX. You need to update to play this map. Visit www.keeperklan.com "
+msgstr "Vous avez une ancienne version de KeeperFX. Vous devez mettre à jour pour jouer cette carte. Visitez www.keeperklan.com "
```

Comment:
Peut être mettre l'adresse du nouveau site officiel de KeeperFX? keeperfx.net à la place de www.keeperklan.com

Reply:
It does not matter, since it is included with the game and that version can play it, this string will never be displayed.